mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -63,10 +63,10 @@ def ocn_modelvsobs(config, field):
     if field.lower() == 'mld':
         obs_filename = "%s/holtetalley_mld_climatology.nc" % obsdir
 
-        ds = xr.open_mfdataset(infiles, preprocess=lambda x: preprocess_mpas(x, yearoffset=yr_offset,
-					   timeSeriesStats=True,
-                               timestr='time_avg_daysSinceStartOfSim',
-                               onlyvars=['time_avg_dThreshMLD']))
+        ds = xr.open_mfdataset(infiles, preprocess=lambda x: 
+                               preprocess_mpas(x, yearoffset=yr_offset,
+                                               timestr=['xtime_start', 'xtime_end'],
+                                               onlyvars=['time_avg_dThreshMLD']))
         ds = remove_repeated_time_index(ds)
```

Comment:
I wasn't able to find one either @xylar.  The way you have done this is much clearer so this is great forward progress we all should be very happy about.  Thanks for taking another look.